tex/generic/pgf/utilities/pgfkeys.code.tex

```diff
@@ -145,7 +152,7 @@
 %
 % \pgfkeyslet{/algo/swap}{\myswap}
 
-\def\pgfkeyslet#1#2{%
+\long\def\pgfkeyslet#1#2{%
   \expandafter\let\csname pgfk@#1\endcsname#2%
 }
```

Comment:
To support `\pgfkeyslet{/foo}{\par}` and `\pgfkeysgetvalue{/foo}{\par}`, these two macros could have `\long`. (sorry this is a not-so-useful comment)